site/blog_collections/_running/2025-12-31-2025_running_year_in_review.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "2025 Running Year in Review"
+description: "A summary of my running in 2025"
+date: 2025-12-31 17:00:00 +0100
+comments: true
+tags: running ultrarunning trailrunning itra utmb review
+draft: true
+# TODO: Upload strava1.jpg and strava2.jpg to site/img_running/2025-12-31/ before merging
+---
+
+This is a summary of how 2025 turned out to be for me in terms of running 🏃‍♂️. I have kept it very short when it comes to mentioning the races etc., otherwise it would be a neverending story 😉.
+
+# Training Volume
+According to Strava, in 2025 I ran `4850K` over `464h20m` with a total elevation gain landing on `36.5K`. That's equivalent to an average of `93K` / `8.9h` per week.
```

Comment:
Update to: 4934 kilometers. 471 hours 23 minutes. 36730 m EG. Re calculate what that means per week on average